BTCPayVsOthers.md

```diff
@@ -90,21 +91,21 @@ Many payment processors claim to have no middleman. Saying the funds go directly
 * If the payment processor has terms and conditions stating they can hold or freeze your account.
 * Fees for using the payment processor are automatically taken out from the customer's payment to the merchant.
 
-Payment processors perform some middleman actions by using a **custodial wallet**. A payment processor can use an internal custodial wallet for altering customer payments before routing them to merchants. This is how they can collect fees, hold payments for verification and processing, etc. This type of wallet is an intermediary between the merchant wallet and the customer wallet. It's the middleman wallet.
+Payment processors are able act as a middleman by using a **custodial wallet**. A payment processor can use an internal custodial wallet for altering customer payments before routing them to merchants. This is how they can collect fees, hold payments for verification and processing, etc. This type of wallet is an intermediary between the merchant wallet and the customer wallet. It's the middleman wallet.
```

Comment:
Payment processors are able to act as a middleman

Reply:
could change to "Payment processors are able to act as middlemen"